client/components/feature-modules/blocks/components/render/list/list.container.tsx

```
@@ -16,24 +17,25 @@ interface Props extends ChildNodeProps {
-    const { getBlock, removeBlock, insertBlock } = useBlockEnvironment();
+    const { getBlock } = useBlockEnvironment();
+    const { removeBlockWithCommand, addBlockWithCommand } = useCommandEnvironment();
 
     const node = getBlock(blockId);
     if (!node) return children;
     const { block } = node;
     const { organisationId, type } = block;
 
     // Create callback handlers for block toolbar
-    const handleDelete = useCallback(() => removeBlock(blockId), [removeBlock, blockId]);
+    const handleDelete = useCallback(() => removeBlockWithCommand(blockId), [removeBlockWithCommand, blockId]);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)